src/main/resources/swagger.api/mod-search.yaml

```
@@ -45,6 +59,20 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/folioCreateIndexResponse'
+        '400':
+          description: Error response when request tries to create index that already exists
+          content:
+            application/json:
+              example: examples/indexCreationError.sample
+              schema:
+                $ref: '#/components/schemas/errorResponse'
+        '500':
+          description: Error response when NullPointerException occurred during code execution
```

There might be other possible errors, but not only NPE, should we update description to something:
 ```suggestion
          description: When unhandled exception occurred during code execution, e.g. NullPointerException
```